assets/src/edit-story/components/panels/index.js

```diff
@@ -34,27 +35,29 @@ export { default as ColorPresetPanel } from './colorPreset';
 const BACKGROUND_SIZE_POSITION = 'backgroundSizePosition';
 const BACKGROUND_DISPLAY = 'backgroundDisplay';
 const BACKGROUND_OVERLAY = 'backgroundOverlay';
+const IMAGE_ACCESSIBILITY = 'imageAccessibility';
 const LAYER_STYLE = 'layerStyle';
 const LINK = 'link';
 const TEXT = 'text';
 const SIZE_POSITION = 'sizePosition';
 const STYLE = 'style';
 const TEXT_STYLE = 'textStyle';
-const VIDEO_POSTER = 'videoPoster';
+const VIDEO_ACCESSIBILITY = 'videoAccessibility';
 const PAGE = 'page';
 const NO_SELECTION = 'noselection';
 
 export const PanelTypes = {
   BACKGROUND_SIZE_POSITION,
   BACKGROUND_DISPLAY,
   BACKGROUND_OVERLAY,
+  IMAGE_ACCESSIBILITY,
   SIZE_POSITION,
   STYLE,
   LAYER_STYLE,
   TEXT,
   TEXT_STYLE,
   LINK,
-  VIDEO_POSTER,
+  VIDEO_ACCESSIBILITY,
```

Comment:
There's some shared code between `VideoAccessibilityPanel` and `ImageAccessibilityPanel`. Wouldn't it make sense to have 1 accessibility panel for both?

Reply:
There is only one shared field between the video and image. So for that reason, I didnt see the point, as the complexity added to make it shared outweighs the benefit.